.github/workflows/pr.yaml

```diff
@@ -0,0 +1,33 @@
+name: Test and coverage
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+
+env:
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  test-and-coverage:
+    if: "!contains(github.event.head_commit.message, 'ci: version bump') && github.actor != 'dependabot[bot]'"
+    name: test-and-coverage
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.15.0'
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Unit test
+        run: yarn test
+      # TODO: Uncomment when we can the http-errors from our organization to codeclimate
```

Comment:
I added `CODECLIMATE_TEST_REPORTER_ID` as a Repository Secret. Let me know if it should go somewhere else.